includes/admin/tools/export/class-give-export-donations.php

```diff
@@ -196,6 +196,21 @@ public function html() {
 											       id="give-export-donation-id"><?php _e( 'Donation ID', 'give' ); ?>
 										</label>
 									</li>
+
+									<?php
+									if ( give_is_setting_enabled( give_get_option( 'sequential-ordering_status', 'disabled' ) ) ) {
+										?>
+										<li>
+											<label for="give-export-seq-id">
+												<input type="checkbox" checked
+												       name="give_give_donations_export_option[seq_id]"
+												       id="give-export-donation-id"><?php _e( 'Payment Number', 'give' ); ?>
```

Comment:
@raftaar1191 This must say `Donation Number`

Reply:
Updated @ravinderk